test/nodes/test_prompt_node.py

```diff
@@ -51,6 +51,14 @@ def test_prompt_templates():
     assert p.prompt_text == "Here is some fake template with variable $baz"
 
 
+def test_prompt_template_repr():
+    p = PromptTemplate("t", "Here is variable $baz")
+    desired_repr = "PromptTemplate(name=t, prompt_text=Here is variable $baz, prompt_params=['baz'])"
+    assert p.__repr__() == desired_repr
```

Comment:
No need to fix this, but just FYI you can also use the built-in method instead of the magic one. Looks a bit better in code :slightly_smiling_face: 

```suggestion
    assert repr(p) == desired_repr
```